### PR TITLE
add instructions to build all-in-one containerized microshift

### PR DESCRIPTION
**Pull request by rootfs**

### CODE REVIEW

**Comment by rootfs** [comment]
@copejon any idea why CI failed?
```
Replaced Dockerfile FROM image registry.access.redhat.com/ubi8/ubi
Caching blobs under "/var/cache/blobs".
Pulling image builder ...
Warning: Pull failed, retrying in 5s ...
Warning: Pull failed, retrying in 5s ...
Warning: Pull failed, retrying in 5s ...
error: build error: failed to pull image: After retrying 2 times, Pull image still failed due to error: errors:
denied: requested access to the resource is denied
unauthorized: authentication required
```

**Comment by copejon** [comment]
@rootfs I'm getting the same errors in my rehearsal PRs. When I figure out why it's suddenly failing, I'll let you know.

File: hack/all-in-one/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,13 @@
+FROM fedora:33
```
Comment (copejon) [inline hack/all-in-one/Dockerfile:1]: Should this be the rhel8 ubi rather than fedora?

Comment (rootfs) [reply hack/all-in-one/Dockerfile:1]: I cannot find instructions how to install crio on ubi

**Pull request merged**